### ISSUE TITLE
NTR: HTLV-1 antibodies present in blood or CSF (Lab: bacteriology, virology, parasitology, fungal)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8071]
New term request HTLV-1 antibodies present in blood or CSF (perlma_110128042508): Human T-lymphotropic virus type I; cerebrospinal fluid

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000730
- label: Positive CSF human T-lymphotropic virus type I antibody positivity
- parents: [Positive CSF pathogen-specific antibody positivity (HP:0430139)](https://hpo.jax.org/app/browse/term/HP:0430139)
- pmid: 
- def: The presence of antibodies in the cerebrospinal fluid (CSF) that react against a component of human T-lymphotropic virus type I.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8071
- synonyms: Positive CSF HTLV-1 antibody positivity(NONE)
